tests/demo_tests/test_ip_v4v6_raw_capture.py

```diff
@@ -0,0 +1,196 @@
+import pytest
+import snappi
+
+
+@pytest.fixture
+def app_intf(utils):
```

Comment:
why aren't we just using `api` (from conftest). although this change is indeed needed.
```python
if getattr(api, 'assistant', None) is not None:
```

Reply:
addressed